src/components/TopBar.js

```diff
@@ -200,6 +128,94 @@ export default class TopBar extends Component {
     )
   }
 
+  buildSignedInOptions() {
+    const { userMenuOpen } = this.state
+
+    return h(Fragment, [
+      h(Collapse, {
+        defaultHidden: true,
+        showIcon: false,
+        animate: true,
+        expandTitle: true,
+        style: styles.nav.profile(false),
+        buttonStyle: { marginBottom: 0 },
+        title: [
+          h(Clickable, {
+            style: { ...styles.nav.item, ...styles.nav.profile(userMenuOpen), boxShadow: `inset ${Style.standardShadow}` },
+            hover: styles.nav.profile(true),
+            onClick: () => this.setState({ userMenuOpen: !userMenuOpen })
+          }, [
+            div({ style: styles.nav.icon }, [
+              profilePic({ size: 32 })
+            ]),
+            div({ style: { whiteSpace: 'nowrap', overflow: 'hidden', textOverflow: 'ellipsis' } }, [
+              getUser().name
+            ]),
+            div({ style: { flexGrow: 1 } }),
+            icon(`angle ${userMenuOpen ? 'up' : 'down'}`,
+              { size: 18, style: { flex: 'none' } })
+          ])
+        ]
+      }, [
+        h(MenuButton, {
+          as: 'a',
+          href: Nav.getLink('profile'),
+          style: styles.nav.profileItem(false),
+          hover: styles.nav.profileItem(true),
+          onClick: () => this.hideNav() // In case we're already there
+        }, [
+          icon('user', { style: styles.nav.icon }), 'Profile'
+        ]),
+        h(MenuButton, {
+          as: 'a',
+          href: Nav.getLink('groups'),
+          style: styles.nav.profileItem(false),
+          hover: styles.nav.profileItem(true),
+          onClick: () => this.hideNav() // In case we're already there
+        }, [
+          icon('users', { style: styles.nav.icon }), 'Groups'
+        ]),
+        h(MenuButton, {
+          onClick: signOut,
+          style: styles.nav.profileItem(false),
+          hover: styles.nav.profileItem(true)
+        }, [
+          icon('logout', { style: styles.nav.icon }), 'Sign Out'
+        ])
+      ]),
+      h(Clickable, {
+        as: 'a',
+        style: styles.nav.item,
+        hover: { backgroundColor: colors.darkBlue[1] },
+        href: Nav.getLink('browse-data'),
+        onClick: () => this.hideNav()
+      }, [
+        div({ style: styles.nav.icon }, [
+          icon('browse', { size: 24 })
+        ]),
+        'Browse Data'
+      ]),
+      div({ style: styles.nav.item }, [
+        div({ style: styles.nav.icon }, [
+          icon('search', { size: 24 })
+        ]),
+        'Find Code', comingSoon
+      ]),
+      h(Clickable, {
+        as: 'a',
+        style: styles.nav.item,
+        hover: { backgroundColor: colors.darkBlue[1] },
+        href: Nav.getLink('workspaces'),
+        onClick: () => this.hideNav()
+      }, [
+        div({ style: styles.nav.icon }, [
+          icon('workspace', { className: 'is-solid', size: 24 })
+        ]),
+        'See All Workspaces'
```

Comment:
I thought we wanted to show at least some of these items while logged out, as a teaser. Can we verify with product?